spherecluster/von_mises_fisher_mixture.py

```diff
@@ -884,7 +906,7 @@ def transform(self, X, y=None):
         if self.normalize:
             X = normalize(X)
 
-        check_is_fitted(self)
+        check_is_fitted(self, "cluster_centers_")
```

Comment:
Could you remove these changes?  `check_is_fitted` no longer allows for a specified parameter in sklearn 0.22.  

Alternatively, make sure you are working off of the most recent develop commit (I made this change a few days ago).

Reply:
yes, I'll pull the latest changes.

Reply:
I updated sklear to 0.22, now one of the tests fails (`test_estimator_spherical_k_means`)
I don't think it's related to the changes I added to `test_vmf_log_detect_breakage` as you suggested

Reply:
You'll need 0.22 and the latest develop branch (which updates things for 0.22)